tools/topology/topology2/cavs/cavs-rt5682-rt1019.conf

```diff
@@ -0,0 +1,250 @@
+<include/common/vendor-token.conf>
+<include/common/tokens.conf>
+<include/common/manifest.conf>
+<include/common/pcm.conf>
+<include/common/pcm_caps.conf>
+<include/common/fe_dai.conf>
+<include/common/route.conf>
+<include/components/host.conf>
+<include/components/dai.conf>
+<include/components/pipeline.conf>
+<include/components/copier.conf>
+<include/dais/ssp.conf>
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+<include/common/data.conf>
+<include/dais/hw_config.conf>
+<include/pipelines/cavs/passthrough-playback.conf>
+<include/pipelines/cavs/passthrough-capture.conf>
+
+#
+# List of all DAIs
+#
+# PCM0 ----> SSP0  (Headset - ALC5682)
+# PCM1 <---> SSP1  (Speaker - ALC1019)
+# PCM2 <---- DMIC01 (dmic 48k capture)
```

Comment:
nitpick: Headset is bidirectional and speaker is playback only. So it is better
```
# PCM0 <---> SSP0  (Headset - ALC5682)
# PCM1 ----> SSP1  (Speaker - ALC1019)
```

Reply:
agree with Bard!

Reply:
Sorry a mistake, corrected it